assets/js/romo/selected_options_list.js

```diff
@@ -47,87 +47,84 @@ RomoSelectedOptionsList.prototype.doRemoveItem = function(itemValue) {
 
 RomoSelectedOptionsList.prototype.doRefreshUI = function() {
   var itemValues  = this._getItemValues();
-  var uiListElem  = this.elem.find('.romo-selected-options-list-items');
-  var uiItemElems = uiListElem.find('.romo-selected-options-list-item');
-  var uiValues    = uiItemElems.get().map(function(uiItemNode) {
-    return $(uiItemNode).data('romo-selected-options-list-value');
-  });
-  var rmNodes = uiItemElems.get().filter(function(uiItemNode) {
-     return itemValues.indexOf($(uiItemNode).data('romo-selected-options-list-value')) === -1;
+  var uiListElem  = Romo.find(this.elem, '.romo-selected-options-list-items')[0];
+  var uiItemElems = Romo.find(uiListElem, '.romo-selected-options-list-item');
+  var uiValues    = uiItemElems.reduce(Romo.proxy(function(values, uiItemNodeElem) {
```

Comment:
@jcredding why not use `map` here? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map